index.html

```diff
@@ -1563,9 +1562,8 @@ <h3 id="applying">
           </li>
           <li>If <a>Type</a>(<var>value</var>) is not "string":
             <ol>
-              <li>If <a>Type</a>(<var>value</var>) is not "undefined",
-              optionally <a>issue a developer warning</a> that the type is not
-              supported.
+              <li>If <a>Type</a>(<var>value</var>) is not "undefined", <a>issue
+              a developer warning</a> that the type is not supported.
```

Comment:
The current document appears to use "not supported" 14 times and "unsupported" 15 times. I'll switch the ones I introduce to "unsupported" but leave the cleanup for others.